src/utils/string.cpp

```diff
@@ -49,7 +49,7 @@ bool isNonasciiChar(unsigned char c) {
 */
 std::string replaceChars(const std::string &str, bool (* predicate)(unsigned char)) {
 	std::stringstream result;
-	const std::size_t maxC = std::pow(2, sizeof(std::string::value_type) * CHAR_BIT) - 1;
+	const std::size_t maxC = (2 << (sizeof(std::string::value_type) * CHAR_BIT - 1)) - 1;
```

Comment:
It's usually done by shifting 1 and not 2. Basically

1 << 0 = 2^0 = 1
1 << 1 = 2^1 = 2
1 << 2 = 2^2 = 4
and so on...

I guess that's why you put ` - 1` after `CHAR_BIT` there but I think that shifting 1 would be much easier to grasp at the first sight as it is the usual way to calculate powers of 2.